app/src/main/java/org/mozilla/fenix/settings/creditcards/controller/CreditCardEditorController.kt

```diff
@@ -32,6 +32,11 @@ interface CreditCardEditorController {
      * @see [CreditCardEditorInteractor.onSaveButtonClicked]
      */
     fun handleSaveCreditCard(creditCardFields: UpdatableCreditCardFields)
+
+    /**
+     * @see [CreditCardEditorInteractor.onUpdateCreditCard]
+     */
+    fun handleUpdateCreditCard(guid: String, creditCardFields: UpdatableCreditCardFields)
```

Comment:
Tapping on the documentation link doesn't do anything.
I think we also need `import org.mozilla.fenix.settings.creditcards.interactor.CreditCardEditorInteractor`.